aip_common_sensor_launch/launch/hesai_XT32.launch.xml

```diff
@@ -19,6 +19,7 @@
   <arg name="use_shared_container" default="false"/>
   <arg name="use_cuda_preprocessor" default="false"/>
   <arg name="udp_only" default="false"/>
+  <arg name="hires_mode" default="true"/>
```

Comment:
Sorry. I realized XT32 does not have the `hires_mode` parameter. 
Since this line is miss-leading, please remove it.
```suggestion
```

You can find the minimal parameter set in nebula_ros/config. [XT32](https://github.com/tier4/nebula/blob/v0.2.6.2/nebula_ros/config/lidar/hesai/PandarXT32.param.yaml)  , [OT128](https://github.com/tier4/nebula/blob/v0.2.6.2/nebula_ros/config/lidar/hesai/Pandar128E4X.param.yaml)